### ISSUE TITLE
Add options to prevent the "seen" and "typing" indicators from triggering?

### ISSUE THREAD

**Issue by beppe9000** [Enrico204/Whatsapp-Desktop#91]
At browser level individual requests can be blocked or faked with relative ease. Too bad Aluxian's Whatsie is discontinued.

**Comment by Enrico204** [comment]
It might be possible. What do you mean "at browser level individual requests can be blocked"? Do you have any clue how to do this? Thanks :-)

**Comment by beppe9000** [comment]
Typing indicator and seen flags are set after HTTP(S) (or WS) requests that can be suppressed in order to prevent that. By "at browser level" I mean that node/electron should either have an API or library for request filtering like [this facebook messenger example](https://github.com/aluxian/Messenger-for-Desktop/blob/d87ee9ba8f1d7d556a4ef9e00e94ca46a5c2123f/src/scripts/browser/menus/expressions/expr-click.js#L313).

The only thing missing is what urls are to be blocked.

**Comment by beppe9000** [comment]
Or maybe I am wrong and whatsapp's implementation can not be disjoined from the rest of requests... some research is needed.

**Comment by Enrico204** [comment]
Uhm, I need to re-check, but AFAIK all status changes are sent (encrypted) on a WebSocket, and Electron has no way to control it. So, no way to intercept it there.
Maybe we can somehow interact with the web app javascript in order to disable it, but I need time to do some experiments.

**Comment by beppe9000** [comment]
Yeah, if that's so it may be more practical to tamper with it in JS
